e2e/safety_checks_test.go

```diff
@@ -22,12 +22,12 @@ func isNilPtrDerefPanic(err string) bool {
 // TestOpClientKeccak256MalformedInputs tests the NewDAClient from altda by setting and getting against []byte("")
 // preimage. It sets the precompute option to false on the NewDAClient.
 func TestOpClientKeccak256MalformedInputs(t *testing.T) {
-	if !runIntegrationTests || runTestnetIntegrationTests {
-		t.Skip("Skipping test as TESTNET env set or INTEGRATION var not set")
+	if !shouldRunTest(LocalOnlyIntegration) {
+		t.Skip()
```

Comment:
why is this local and not standard?

Reply:
Unknown- I changed it to also run on holesky, and the test is passing. I'm going to leave it like that